src/rnp/rnpcfg.c

```diff
@@ -61,11 +55,18 @@ rnp_cfg_load_defaults(rnp_cfg_t *cfg)
     rnp_cfg_set(cfg, CFG_SUBDIRGPG, SUBDIRECTORY_RNP);
     rnp_cfg_set(cfg, CFG_SUBDIRSSH, SUBDIRECTORY_SSH);
     rnp_cfg_setint(cfg, CFG_NUMTRIES, MAX_PASSPHRASE_ATTEMPTS);
-
-    return RNP_OK;
 }
 
-int
+static void rnp_cfg_get_defkey(rnp_cfg_t *cfg, rnp_params_t *params);
+static bool rnp_cfg_get_ks_info(rnp_cfg_t *cfg, rnp_params_t *params);
+
+/** @brief apply configuration from keys-vals storage to rnp_params_t structure
+ *  @param cfg [in] rnp config, must be allocated and initialized
+ *  @param params [out] this structure will be filled so can be further feed into rnp_init
+ *
+ *  @return true on success, false if something went wrong
```

Comment:
If something went wrong, does user needs to release memory that potentially could be allocated to ``params`` before "something" occured ?